chromium_src/chrome/browser/ui/tab_helpers.cc

```diff
@@ -5,10 +5,6 @@
 
 #include "brave/browser/brave_tab_helpers.h"
 
-// Include this to make sure we don't unintentionally rename NetErrorTabHelper
```

Comment:
does this not apply to RenderFrameHostReceiverSet now?

Reply:
I only that symbol in these files in cr94:
<pre>
$ git grep -l NetErrorTabHelper -- '*.h'
chrome/browser/net/net_error_tab_helper.h
chrome/renderer/net/net_error_helper.h
</pre>

In cr93 it was in 
<pre>
$ git grep -l NetErrorTabHelper -- '*.h'
chrome/browser/net/net_error_tab_helper.h
chrome/renderer/net/net_error_helper.h
content/public/browser/web_contents_receiver_set.h
</pre>